### DISCUSSION TITLE
Updated Docker version

### DISCUSSION THREAD

**Discussion by blubkens** [petrvlcek/bisecur2mqtt#7] [Q&A] [answered]
Hi Petrvlcek,

Is there a way to get the update version in Docker available?
The current docker image uses the double MQTT PASSWORD section which was recently updated.
If you could refresh the Docker image that would be great.
Thanks,

Blubkens!

**Comment by petrvlcek** [comment, marked as answer]
The new image is avaliable, it is released also as version 1.0.0. Please try it and don't hesitate to open an issue if anything doesn't work. Thank you!

**Reply by blubkens** [reply]
Hi Petrvlcek,

I was just about to learn on how to create these myself.
Thanks for your swift reply, i'll pull the new docker and see if that works.
Cheers,

Blubkens.
